game/main.cpp

```diff
@@ -81,7 +81,9 @@ int main(int argc, char** argv) {
 
 #ifndef __AVX2__
   if (get_cpu_info().has_avx2) {
-    printf("Note: your CPU supports AVX2, but this build was not compiled with AVX2 support\n");
+    printf(
```

Comment:
let's just get rid of this print. Nobody cares about the tiny 2% performance increase from the AVX2 version of time of day interpolation anyway.